effects/antidot_effect.py

```diff
@@ -0,0 +1,44 @@
+import message_text_config as msg
+import utils
+from citizens.citizen import Citizen
+from citizens.player import Player
+from game import Game
+
+from effects.effect import Effect
+
+
+class AntidotEffect(Effect):
+    def __init__(self, game: Game, name: str, creator: Citizen) -> None:
+        super().__init__(game, name, creator, -1)
+
+    def _activate_impl(self) -> bool:
+        target_number = utils.read_target_number(
+            self.context, msg.AntidoteMessages.ACTIVATION_CHOOSE_TARGET,
+            self._validate)
+        self.targets.append(self.game.citizens[target_number - 1])
+
+        self.user_interaction.save_active(
+            msg.AntidoteMessages.ACTIVATION_SUCCESS.format(
+                self.targets[0].name))
+
+        return True
+
+    def _resolve_impl(self) -> bool:
+
+        if self.activation_round == self.game.round_number:
+            self.targets[0].hp += 1000
+            return False
+
+        if (self.activation_round + 1) == self.game.round_number:
```

Comment:
I'd prefer to swap comparable values.
Because we are comparing current round number TO some value. Not some value to.round number

Reply:
Done